react/src/lib/components/DeckGLMap/components/Map.tsx

```diff
@@ -298,6 +298,15 @@ function adjustCameraTarget(
     return vs;
 }
 
+function checkIfViewsIsDefined(views: ViewsType | undefined) {
```

Comment:
I think this does the same as `lodash.isEmpty()` function. Could we replace it?

Reply:
tested and it seems work for both empty or undefined cases. Have replaced it in the new push